packages/steveo/src/common.ts

```diff
@@ -157,6 +157,7 @@ export interface IRegistry {
   emit(name: string, ...args: any): void;
   addTopic(name: string, topic?: string): void;
   getTask(topic: string): RegistryElem | null;
+  count(): number;
```

Comment:
Not sure I understand the value in exposing a `count` on the registry given we get `getTopics` - for testing we could count the returned topics, but we can also assert that the topics are correctly prefixed (assuming I've understood topics correctly)

Reply:
You understood correctly, @sugendran. 
To test the prefixed topics I check whether the topic was registered as expected (with prefix, camelcase, etc) and the `count` method has absolutely nothing to do with it.

I just added the method `this.registry.count()` because IMO it reads better, and is easier to remember than `this.registry.getTopics().length`